### DISCUSSION TITLE
new bypass

### DISCUSSION THREAD

**Discussion by Bunjoli0** [3kh0/ext-remover#835] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
step 1. get a part time job with parent permission, get a work permit if ur under 16. 
step 2. Save money.
Step 3. Buy a laptop or Chromebook of your own.
Step 4. No more block

**Comment by DankFireR** [comment]
hardy har har
yeah im p sure most people use it in school, not outside of it
idk if all schools allow a personal computer

**Comment by 3kh0** [comment]
Not even a chromebook, just a good laptop

**Comment by rwg100** [comment]
better idea!
buy a bomb (aka a chromebook made 2022 to present)
